src/web/components/Input/Input.scss

```diff
@@ -10,10 +10,10 @@
     border: 1px solid var(--theme-input-border);
     border-radius: 5px;
     padding: 0.5rem;
+    margin-top: 0.5rem;
```

Comment:
This (and the below change) could affect a lot of spots that we've tweaked to get just right - have you had a look at a few different places to make sure the change looks right?

Reply:
Yeah I've gone through all the components that are built on the underlining Input component and checked formatting and added it where necessary.